model/marine/marine_background_error_hybrid_diffusion_diffusion.yaml.j2

```diff
@@ -57,48 +57,50 @@ components:
   weight:
     value: 1.00
 
+{% if marine_ensemble_b | default(false) %}
 - covariance:
-    covariance model: SABER
```

Comment:
are the indentation changes here intentional?

Reply:
Probably not. Checking ...